pkg/features/features.go

```diff
@@ -41,6 +41,14 @@ const (
 	//
 	// Enable usage of Provision of PVCs from snapshots in other namespaces
 	CrossNamespaceVolumeDataSource featuregate.Feature = "CrossNamespaceVolumeDataSource"
+
+	// TODO: What are the appropriate owners here? Never implemented a feature flag before, did my best guess
+	// owner: @sunnylovestiramisu @ConnorJC3
+	// kep: https://kep.k8s.io/3751
+	// alpha: v1.29
+	//
+	// Pass VolumeAttributesClass parameters to supporting CSI drivers during CreateVolume
+	VolumeAttributesClass featuregate.Feature = "VolumeAttributesClass"
```

Comment:
The question here is also do we want a diff feature gate for the sidecars from the k8s feature gate. @msau42